ablog/__init__.py

```diff
@@ -88,7 +89,7 @@ def config_inited(app):
 
 
 def builder_inited(app):
-    if app.config.skip_injecting_base_ablog_templates:
+    if isinstance(app.builder, LaTeXBuilder) or app.config.skip_injecting_base_ablog_templates:
```

Comment:
```
not (isinstance(app.builder, HTMLBuilder) or app.config.skip_injecting_base_ablog_templates))
```

We probably only want blog posts in HTML builds?

Reply:
Good point.